src/phoenix/core/model_schema_adapter.py

```diff
@@ -14,6 +15,12 @@
 
 
 def create_model_from_datasets(*datasets: Optional[Dataset]) -> Model:
+    # TODO: move this validation into model_schema.Model.
```

Comment:
probably best to file a ticket on this one and put in the TODO so it doesn't get lost? I think running the getter to run the validation might confuse some